### DISCUSSION TITLE
I am encountering a fatal error when trying to obtain a download from a previous version of NMS

### DISCUSSION THREAD

**Discussion by apandemic** [SteamRE/DepotDownloader#253] [Q&A]
The title says it all. I am using a prompt to download a version that's supposed to work. Here is an example of the string of code that I am using. 

DepotDownloader -app 275850 -depot 275851 -manifest 8262658978126728861 -dir "no_mans_sky_v1.38" -username UN -password PW

Looking at the readme page that your program is attached to. You recommend using the < > around the password and Username. I'm confused if that's to indicate where to place your username or passwords, or if its vital for the code to work. I  am also wondering if it has something to do with where the file has been placed. I have placed the file in OS(C;). Since that is what the tutorial told me to do. I could be wrong however. 

What is the best course of action to fix this problem 

Here is what the error looks like 

C:\depotdownloader> DepotDownloader -app 275850 -depot 275851 -manifest 8262658978126728861 -dir "no_mans_sky_v1.38" -username apandemic -password sillystar155
A fatal error occurred. The required library hostfxr.dll could not be found.
If this is a self-contained application, that library should exist in [C:\depotdownloader\].
If this is a framework-dependent application, install the runtime in the global location [C:\Program Files\dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime location in [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x64\InstallLocation].

**Comment by psychonic** [comment]
It seems that you don't have the .NET Core Runtime installed.
